app/views/customers/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1><%= @customer.name %></h1>
+
+<h2> Items: </h2>
+<ul>
+    <% @customer.items.each do |item| %>
+        <li>
+            <%= item.name %>
+            $<%= item.price %>
+            <%= item.supermarket.name %>
+        </li>
+    <% end %>
+</ul>
+
+<h2> Add Item: </h2>
+<%= form_with url: "/customers/#{@customer.id}/items", method: :post do %>
+    <div>
+        <%= label_tag :item_id, "Item ID" %>
+        <%= text_field_tag :item_id %>
+    </div>
+    <%= submit_tag "Add Item" %>
```

Comment:
Wrong syntax for your form elements since you're using a `form_with`. Review the form lesson plan for the correct syntax.